RELEASING/release-notes-4-0/README.md

```diff
@@ -0,0 +1,151 @@
+
+<div>
+    <img src="media/tags-1.png" alt="Image" width="100%">
+    <img src="media/tags-2.png" alt="Image" width="100%">
+</div>
+
+### New CHANGELOG format
+
+We changed the structure of the `CHANGELOG.md` file in [#26800](https://github.com/apache/superset/pull/26800) to better organize the contents of each release and also to deal with GitHub size limitations when displaying the file. Now every release will have its own file at `CHANGELOG/<version>.md`. The main `CHANGELOG.md` file is now an index with links to all releases.
+
+### Improved drag and drop experience when editing a dashboard
+
+When a component was being dragged towards the edge of the tab container or the row/column containers, multiple drop indicators were often displayed. This created confusion about the exact insertion point of the element. To fix this, we built in [#26699](https://github.com/apache/superset/pull/26699) and [#26313](https://github.com/apache/superset/pull/26313) a distinct, non-conflicting area for the drop zone, which is highlighted during the dragging process to clearly indicate where the element will be placed. We also improved the forbidden drop zones to prevent users from dropping elements in invalid locations.
+
+<div>
+    <img src="media/dashboard-dnd-1.png" alt="Image" width="100%">
+    <img src="media/dashboard-dnd-2.png" alt="Image" width="100%">
+</div>
+
+### Improved drag and drop experience when editing a chart
+
+Now, during dragging, all droppable zones are highlighted, with distinct colors indicating available and unavailable drop locations. This enhancement clarifies potential drop points and helps avoid inadvertent placements in invalid areas. The update also aligns the drag-over feedback with the dashboard's drag-and-drop modifications, ensuring a uniform and enhanced user experience.
+
+![Drag and drop](media/explore-dnd.png)
+
+### Dropping support for 3.0.X versions
+
+In accordance with our [release process](https://github.com/apache/superset/wiki/Release-Process), we are dropping support for the 3.0.X versions. As a result, we will no longer be providing bug fixes for these versions. We strongly recommend that all users upgrade to the latest version to take advantage of the newest features and bug fixes. Moving forward, the supported versions will be 3.1.X and 4.0.X. Bug fixes will continue to be backported to 3.1.X until the next minor release. For more information, please refer to our [release schedule](https://github.com/apache/superset/wiki/Release-Process#schedule).
+
+### Feature flag changes
+
+Following our 4.0 proposals, the following feature flags were removed:
```

Comment:
```suggestion
Following our 4.0 proposals, the following features were permanently enabled and their flags removed:
```

Reply:
That's inaccurate. Some flags were removed because the feature does not exist anymore.

Reply:
Ah.  What I was trying to address was if someone reads this and thinks these _features_ were removed, when in fact many were permanently enabled.  I'll try again.

Reply:
I changed my suggestion - if you think that confuses more than it helps, feel free to ignore and mark resolved.

Reply:
Suggestion accepted.